portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Adetowo Ademiluyi's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+    <h1 id="myHeader" font-size: 50px>PROJECTS</h1>
+    <p>Open this PDF file to see my resume <a href="new_images/resume.jpg">example</a>.</p>
+    <div id="project_body">
+      <h1>TIC TAC TOE</h1>
+      <img src="new_images/tic-tac-toe.jpg"/>
+      <p>
+        • Developed a Tic Tac Toe game in replit that
```

Comment:
Use < li > (after removing the spaces). Do the same things for the following items.